content/pt-br/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -16,124 +20,324 @@
-
             </div>
-            <div class="col-md-4">
-                <div class="content__box content__box_lined">
-                    <h3>Sumário:</h3>
-                    <ul>
-                        <li>Pods</li>
-                        <li>Nós (Nodes)</li>
-                        <li>Principais comandos do Kubectl</li>
-                    </ul>
-                </div>
-                <div class="content__box content__box_fill">
-                        <p><i>
-                            Um Pod é um grupo de um ou mais contêineres de aplicativos (como Docker) que inclui armazenamento compartilhado (volumes), endereço IP e informações sobre como executá-los.
-                        </i></p>
-                </div>
+            <div class="content__box content__box_fill">
+                    <p><i>
+                      Um Pod é um grupo de um ou mais contêineres de
+                      aplicativos (como Docker) que inclui armazenamento
+                      compartilhado (volumes), endereço IP e informações
+                      sobre como executá-los.
+                    </i></p>
             </div>
         </div>
-        <br>
+      </div>
+      <br>
 
-        <div class="row">
-            <div class="col-md-8">
-                <h2 style="color: #3771e3;">Visão geral sobre os Pods</h2>
-            </div>
+      <div class="row">
+        <div class="col-md-8">
+          <h2 style="color: #3771e3;">Visão geral sobre os Pods</h2>
         </div>
+      </div>
 
-        <div class="row">
-            <div class="col-md-8">
-                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_03_pods.svg"></p>
-            </div>
+      <div class="row">
+        <div class="col-md-8">
+          <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_03_pods.svg"></p>
         </div>
-        <br>
+      </div>
+      <br>
 
-        <div class="row">
-            <div class="col-md-8">
-                <h2>Nós (Nodes)</h2>
-                <p>Um Pod sempre será executando em um <b>Nó</b>. Um Nó é uma máquina de processamento em um cluster Kubernetes e pode ser uma máquina física ou virtual. Cada Nó é gerenciado pelo Control Plane. Um Nó pode possuir múltiplos Pods e o Control Plane do Kubernetes gerencia automaticamente o agendamento dos Pods nos nós do cluster. Para o agendamento automático dos Pods, o Control Plane leva em consideração os recursos disponíveis em cada Nó.</p>
+      <div class="row">
+        <div class="col-md-8">
+          <h2>Nós</h2>
+          <p>
+            Um Pod sempre será executando em um <b>Nó</b>. Um Nó é uma
```

Comment:
Hi @stormqueen1990, please, can you fix this typo in that word "executando" and change to "executado".
Thanks

Reply:
Just fixed this!